lib/matplotlib/axis.py

```diff
@@ -802,6 +802,7 @@ def clear(self):
         """
 
         self.label.set_text('')  # self.set_label_text would change isDefault_
+        self.label.set_fontsize(mpl.rcParams['font.size'])
```

Comment:
I suppose we should create a new Text instance here. There are a lot more parameters that could have been set and would need resetting.

Reply:
Thank you for your review. I pushed a new commit. A Text instance is created in `clear` and default values are set.